README.md

```diff
@@ -67,23 +114,26 @@ before = "main.lua"
 name = "nativefs"
 ```
 
-### Patch variants
+### TL;DR - Patch variants
 
-This file contains two patch definitions - a pattern patch, which (currently) changes a single line at a position offset to some pattern match, and a copy patch, which reads one or more input lua files and either appends or prepends them onto the target. The former is used when you need to surgically embed code at specific locations in the target (very useful for modloader init routines), and the latter is designed for use when you need to bulk inject position-independent code into the game.
+- Use `pattern` patches are used to surgically embed code at specific locations within the target. Supports the `*` and `+` wildcards.
```

Comment:
```suggestion
- Use `pattern` patches to surgically embed code at specific locations within the target. Supports the `*` and `+` wildcards.
```

Reply:
Which wildcards are supported again? A comment said `*` and `?`

Reply:
Good catch, it's supposed to be `*` and `?`.